### DISCUSSION TITLE
Billing question

### DISCUSSION THREAD

**Discussion by AdrianDutu** [codemagic-ci-cd/codemagic-docs#1896] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Hello,

This is not an issue but a question about billing.

So on the individual plan: it says 500 min free per month and extra are available with pay as you go pricing. 
So now if i'm looking at the teams: there's no mention of the free 500 mins but a $0.045 / minute for Linux machines. Does this kick in after the free 500 or those ones are available only for the personal account?

Do people need to add their credit card details to join a team and have access to the applications in that team?
I saw several reports that this was the case, which would be a total deal breaker.

Regards,
Adrian

### Expected results

N/A

### Actual results

N/A

**Comment by kulakarla** [comment]
Hello @AdrianDutu 

Free minutes are only for the personal account. For team builds, the minute cost is immediate.

No, people don't have to add their credit cards in order to join a team. They just need to accept the invitation.
Whoever is creating the team though in the first place, must add card details though.

**Comment by kulakarla** [comment]
I'm tenatively closing this thread due to inactivity.
